resources/css/launcher.scss

```diff
@@ -265,3 +265,16 @@ body {
 .me_32px {
     margin-right: 32px;
 }
+
+.app-entry-highlight {
+    animation: appWasInstalled 3s;
+}
+
+@keyframes appWasInstalled {
+    0% {
+        background-color: #c8e6c9;
+    }
+    100% {
+        background-color: white;
+    }
+}
```

Comment:
Please do not put more stuff into the global `launcher.scss`. Instead define an own SCSS file for the affected component.

Reply:
> Should the animation also appear on deinstallation? For me it sometimes does and sometimes not. Maybe depending on whether the app was already installed when the launcher was started?

I don't feel like it should. Normally an uninstalled app becomes irrelevant for you.
The exception is during a reinstallation, which is not something a user is expected to do (except as a workaround for bugs)

Reply:
> Before I also considered animating the movement of the app from the old to the new position but you solution is less effort and good enough for now.

I considered it for a bit and I think it would be possible to do a smooth scroll animation if we change the keys. Currently enabling a smooth transition doesn't work well because chrome by default will jump to the item and `.scrollIntoView()` guarantees that it's at the top (also makes sure that this behaviour happens if you move after installing. A case which you described to me earlier).